testsuite/MDAnalysisTests/core/test_accumulate.py

```diff
@@ -149,3 +149,121 @@ def test_total_mass_duplicates(self, group):
         with pytest.warns(DuplicateWarning) as w:
             assert_almost_equal(group2.total_mass(), ref)
             assert len(w) == 1
+
```

Comment:
Can you make a test where the dipole and / quadrupoles should be exactly zero by virtue of symmetry?
This would be a good test for essential correctness.

Reply:
@jaclark5 just this and I think Im pretty happy with this. I'll leave it to @orbeckst to have a look at the notation as discussed in the thread above.

Reply:
Updated methane coordinates to have perfect symmetry and fulfill this desired role.